apiserver/facades/client/spaces/shims.go

```diff
@@ -3,33 +3,22 @@
 
 package spaces
 
-import (
-	"github.com/juju/errors"
-
-	"github.com/juju/juju/state"
-)
-
 // stateShim forwards and adapts state.State
 // methods to Backing methods.
 type stateShim struct {
-	*state.State
 }
 
 // NewStateShim returns a new state shim.
-func NewStateShim(st *state.State) (*stateShim, error) {
-	return &stateShim{
-		State: st,
-	}, nil
+func NewStateShim() (*stateShim, error) {
+	return &stateShim{}, nil
 }
 
 func (s *stateShim) ConstraintsBySpaceName(spaceName string) ([]Constraints, error) {
-	found, err := s.State.ConstraintsBySpaceName(spaceName)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	cons := make([]Constraints, len(found))
-	for i, v := range found {
-		cons[i] = v
-	}
-	return cons, nil
+	// TODO: This must be implemented using the domain services.
+	return nil, nil
+}
+
+func (s *stateShim) IsController() bool {
+	// TODO: This must be implemented using the domain services.
```

Comment:
This should be as easy as checking if the `IsControllerModelScoped` on the facade context.